docs/docs/References/Glossary.md

```diff
@@ -72,7 +72,7 @@ For example, a MySQL plugin might provide an operation called "stop" which knows
 The process of making a virtual copy of a dataset and making it available for use on a target environment.
 
 ## Replication
-Delphix allows engine user to replicate data objects between Delphix Engines by creating a replication spec. Data objects that belong to a plugin can also be part of the replication spec. More details [here](/Best_Practices/Replication/Replication.md).
+Delphix allows end users to replicate data objects between Delphix Engines by creating a replication profile. Data objects that belong to a plugin can also be part of the replication profile. Refer to the [Delphix Engine Documentation](https://docs.delphix.com/docs/data-backup-and-recovery-solutions/delphix-replication) for more details.
```

Comment:
do we not want to reference anywhere in this code about replication? since the other one also links to delphix docs, does it make more sense for them to be linked to the more plugin author relevant docs?

Reply:
oh also I think we had a greed to just link high level docs until we figure out if there was a way to know when a link is broken?

Reply:
yup, I forgot to change this one to the top level docs, fixed.